packages/filesystem/src/node/encoding.ts

```diff
@@ -0,0 +1,48 @@
+/********************************************************************************
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+// copied from vscode: https://github.com/Microsoft/vscode/blob/master/src/vs/base/node/encoding.ts
```

Comment:
Another CQ?

Reply:
It's from the same upstream project, so a single CQ covering both files is sufficient. 

@vince-fugnitto a committer has to open the CQ - would you please take care of it? See these [instructions](https://github.com/theia-ide/theia/wiki/Registering-CQs#case-3rd-party-project-code-copiedforked-from-another-project-into-eclipse-theia-maintained-by-us) and let me know if you have questions.

Reply:
@vince-fugnitto @marcdumais-work Hi! Is there anything that I need to do?

Reply:
@a1994846931931 I'll have to open a CQ for you so we can continue with the review :)

Reply:
The CQ was created and is linked in the PR's description.